src/components/DataGridWithInfiniteScroll/DatagridInfiniteScrollMockData.tsx

```diff
@@ -0,0 +1,643 @@
+                  "productId": "ST8000NM02052FF",
+                  "manufacturer": "ATA",
+                  "partNumber": "TH092WDWGJHDGKFHHKJH",
+                  "SASAddress": "0x87694853ACDB",
+                  "powerStatus": "Spun Up",
+                  "failurePredicted": "No",
+                  "deviceDescription": "Disk 1 of 120 blockplane integrated",
+                  "capacityAvailable": "200 GiB"
+                },
+                {
+                  "status": "good",
+                  "name": "Disk 1",
+                  "slot": 1,
+                  "serial": "LPQW7891",
+                  "type": "Read/Write",
+                  "capacityUsed": "100 GiB",
+                  "controller": 1,
+                  "blockSize": "512 bytes",
+                  "operationalState": "N/A",
+                  "remainingDriveLife": "N/A",
+                  "progress": "N/A",
+                  "usedRAIDDiskSpace": "0 GB",
+                  "availableRAIDDiskSpace": "0 GB",
+                  "negotiatedSpeed": "6 Gbps",
+                  "capableSpeed": "6 Gbps",
+                  "serialNumber": "ZAK1CKG2d",
+                  "revision": "PB53",
+                  "productId": "ST8000NM02052FF",
+                  "manufacturer": "ATA",
+                  "partNumber": "TH092WDWGJHDGKFHHKJH",
+                  "SASAddress": "0x87694853ACDB",
+                  "powerStatus": "Spun Up",
+                  "failurePredicted": "No",
+                  "deviceDescription": "Disk 1 of 120 blockplane integrated",
+                  "capacityAvailable": "200 GiB"
+               
+                }
+          
+          
+    ]}
+
+    export const issueData = {
+        node:[
+        {
+          "acknowledged": false,
+          "clearType": "Auto",
+          "id": "SXNzdWU6ZGVja3MtYXRsYW50aWMtVXBkYXRlLUFwcGxpY2F0aW9uLWF0bGFudGljLWRlY2tz",
+          "message": "This is a message for update of the application",
+          "namespace": "atlantic",
+          "reason": "Update",
+          "remedies": ["Restart your cluster!", "Delete your cluster and nuke it!"]
+        },
+        {
+            "acknowledged": false,
+            "clearType": "Auto",
+            "id": "SXNzdWU6ZGVja3MtYXRsYW50aWMtVXBkYXRlLUFwcGxpY2F0aW9uLWF0bGFudGljLWRlY2tz",
+            "message": "This is a message for update of the application",
+            "namespace": "atlantic",
+            "reason": "Update",
+            "remedies": ["Restart your cluster!", "Delete your cluster and nuke it!"]
+          },
+          {
+            "acknowledged": false,
+            "clearType": "Auto",
+            "id": "SXNzdWU6ZGVja3MtYXRsYW50aWMtVXBkYXRlLUFwcGxpY2F0aW9uLWF0bGFudGljLWRlY2tz",
+            "message": "This is a message for update of the application",
+            "namespace": "atlantic",
+            "reason": "Update",
+            "remedies": ["Restart your cluster!", "Delete your cluster and nuke it!"]
```

Comment:
still see duplicate data in the mock file. Why are we not using simply for loop?

Reply:
Done